--- /opt/chef-server/embedded/cookbooks/chef-server/templates/default/erchef.config.erb.orig  2015-03-10 17:50:25.000000000 +0000
+++ /opt/chef-server/embedded/cookbooks/chef-server/templates/default/erchef.config.erb 2015-03-11 11:44:06.000000000 +0000
@@ -16,9 +16,8 @@
               {lager_file_backend, [
                                     {file, "<%= File.join(@log_directory, 'erchef.log') %>"},
                                     {level, info},
-                                    {size, <%= @log_rotation['file_maxbytes'] %>},
-                                    {date, "$D0"},
-                                    {count, <%= @log_rotation['num_to_keep'] %>},
+                                    {size, 0},
+                                    {date, ""},
                                     {formatter_config, [date, " ", time, " [", severity, "] ", message, "\n"]}
                                    ]}
               ]},
@@ -32,16 +31,12 @@

           %% Maximum size of the crash log in bytes, before its rotated, set
           %% to 0 to disable rotation - default is 0
-          {crash_log_size, <%= @log_rotation['file_maxbytes'] %>},
+          {crash_log_size, 0},

           %% What time to rotate the crash log - default is no time
           %% rotation. See the lager README for a description of this format:
           %% https://github.com/basho/lager/blob/master/README.org
-          {crash_log_date, "$D0"},
-
-          %% Number of rotated crash logs to keep, 0 means keep only the
-          %% current one - default is 0
-          {crash_log_count, <%= @log_rotation['num_to_keep'] %>},
+          {crash_log_date, ""},

           %% Whether to redirect error_logger messages into lager - defaults to true
           {error_logger_redirect, true},
